spec/policies/casa_case_policy/casa_case_policy_spec.rb

```diff
@@ -5,19 +5,97 @@
 
   permissions :update_case_number? do
     context 'when user is an admin' do
-      it 'returns true' do
-        casa_case = create(:casa_case)
-        admin = create(:user, :casa_admin)
-        expect(Pundit.policy(admin, casa_case).update_case_number?).to eq true
+      it 'does allow update case number' do
```

Comment:
This test title doesn't fit the test body, case number is not being updated here, just created